apis/pipeline/pipeline.yaml

```diff
@@ -4494,6 +4494,45 @@ components:
                     type: integer
                     example: 5
 
+        AWSAuthConfigMap:
+            type: object
+            properties:
+                data:
```

Comment:
I don't think we need the `data` level here.

Reply:
removed `data` level in cbd6f9bd0